app/scripts/templates/sign_in_recovery_code.mustache

```diff
@@ -0,0 +1,35 @@
+<div id="main-content" class="card">
+  <header>
+    <h1 id="fxa-recovery-code-header">{{#t}}Enter recovery code{{/t}}
+        {{#serviceName}}
+            <span class="service">{{#t}}Continue to %(serviceName)s{{/t}}</span>
+        {{/serviceName}}
+    </h1>
+  </header>
+
+  <section>
+    <div class="error"></div>
+    <div class="success"></div>
+
+    <div class="graphic graphic-recovery-codes"></div>
+
+    <p class="verification-recovery-code-message">
+    {{#t}}Please enter a recovery code that was provided to you during setup.{{/t}}
+    </p>
+
+    <form novalidate>
+      <div class="input-row token-code-row">
+        <input type="text" class="tooltip-below recovery-code" placeholder="{{#t}}Enter 8-digit recovery code{{/t}}" required autofocus />
+      </div>
+
+      <div class="button-row">
+        <button type="submit" class="use-logged-in">{{#t}}Continue{{/t}}</button>
+      </div>
+
+    </form>
+
+    <div class="links">
+        <a id="use-backup-link" class="right" href="/signin_totp_code">{{#t}}Enter security code{{/t}}</a>
```

Comment:
What's the difference between a recovery code and a security code? The wording is confusing, does the security code come from the authenticator app?

In "sign_in_totp_code.mustache" the wording is "Use recovery code". Maybe better wording here is "Use authenticator app"

Reply:
I can see how this is slightly confusing. Security codes are the same as TOTP codes/authenticator app codes/any device that can generate TOTP codes. I'm not sure what the original reason was other than it seems to be a generic and recognizable security term? Recovery codes are the one time use codes if you lose your authenticator app/device.

> Maybe better wording here is "Use authenticator app"

I wonder if it is ok to keep the `Use security code` here since on the login page, it says `Open your authentication app and enter the security code it provides.` or maybe `Back`. @ryanfeeley Thoughts?